typings/index.d.ts

```diff
@@ -2924,6 +2925,13 @@ declare module 'discord.js' {
 
   type TargetUser = number;
 
+  interface TypingData {
```

Comment:
should the `timeout` property also be documented here?

Reply:
Yes, it should. I've pushed the new changes to apply the changes you recommended / pointed out. Thank you for the continuous feedback. :+1: